pokemon_search/src/App.css

```diff
@@ -84,3 +84,86 @@
 .fourth:hover {
   background-position: 0;
 }
+
+.details-page {
```

Comment:
A great thing about CSS module ( i.e. importing CSS files into React components ) is that we can split style based on components. So a common practice in this case would be creating `Details.css` and move things related to Details to that file